mantidproject/mantid
# Corrected manage user directories help page link

**Description of work.**

The help link for the manage user directories dialog was not working because it expected the rst file to be in concepts whereas it was actually in interfaces this fixes this issue.

**To test:**

Build the qt help docs, build target is docs-qthelp. Open the Manage user directories interface and click on the help button in the bottom left. Check that a help page is produced.



Fixes #22132 


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
